### PR TITLE
New package: transmission-remote-gtk

### PR DESCRIPTION
**Pull request by SolitudeSF**

### CODE REVIEW

File: srcpkgs/transmission-remote-gtk/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+pkgname=transmission-remote-gtk
+version=1.3.1
+revision=1
+build_style=gnu-configure
+build_options="geoip"
+hostmakedepends="pkg-config intltool"
+makedepends="$(vopt_if geoip geoip) appstream-glib-devel libcurl-devel gtk+3-devel json-glib-devel libnotify-devel libproxy-devel libmrss-devel"
+short_desc="GTK client for remote management of the Transmission BitTorrent client"
+maintainer="SolitudeSF <solitudesf@protonmail.com>"
+license="GPL-3"
```
Comment (Gottox) [inline srcpkgs/transmission-remote-gtk/template]: This is not GPL-3.

File: srcpkgs/transmission-remote-gtk/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'transmission-remote-gtk'
+pkgname=transmission-remote-gtk
+version=1.3.1
+revision=1
+build_style=gnu-configure
+build_options="geoip"
+hostmakedepends="pkg-config intltool"
+makedepends="$(vopt_if geoip geoip) appstream-glib-devel libcurl-devel gtk+3-devel json-glib-devel libnotify-devel libproxy-devel libmrss-devel"
```
Comment (Gottox) [inline srcpkgs/transmission-remote-gtk/template]: 👍 for using build options 😄

Comment (ebfe) [reply srcpkgs/transmission-remote-gtk/template]: any reason to not enable geoip by default?

**Comment by SolitudeSF** [comment]
added geoip as default and changed license to gpl-2

**Pull request merged**